### ISSUE TITLE
Drop IE-Support?

### ISSUE THREAD

**Issue by fspoettel** [fspoettel/hagrid#45]
Dropping IE-Support would make it possible to use `flex-basis` instead of `width` for grid-items.

**Comment by renatocarvalho** [comment]
What versions of internet explorer would be affected? All versions of IE?

I would consider ok to push this forward and if a support to IE is still needed, people could use a previous version of Hagrid. What do you think?

**Comment by fspoettel** [comment]
@renatocarvalho Yes, all versions including IE 11 which is due to be dropped in April. Edge would ofc be supported.

**Comment by renatocarvalho** [comment]
Cool! I would say that we can think forward anyway and remove IE support from the v4.0.
